coral/cycle_decomposition.py

```diff
@@ -45,25 +42,30 @@ def minimize_cycles(
     model_prefix="pyomo",
     solver_to_use: datatypes.Solver = datatypes.Solver.GUROBI,
 ):
-    """Cycle decomposition by minimizing the number of cycles/paths
-
-    amplicon_id: integer, amplicon ID
-    g: breakpoint graph (object)
-    k: integer, maximum mumber of cycles/paths allowed in cycle decomposition
-    total_weights: float, total length-weighted CN in breakpoint graph g
-    node_order: dict maps each node in the input breakpoint graph to a distinct integer, indicating a total order of the nodes in g
-    pc_list: list of subpath constraints to be satisfied, each as a dict that maps an edge to its multiplicity
-            *** note that all subpath constraints in this list are required to be satisfied ***
-            *** otherwise will return infeasible ***
-    p_total_weight: float between (0, 1), minimum proportion of length-weighted CN to be covered by the resulting cycles or paths,
-                    default value is 0.9
-    p_bp_cn: float float between (0, 1), minimum proportion of CN for each discordant edge to be covered by the resulting cycles or paths,
-                    default value is 0.9
-    num_threads: integer, number of working threads for gurobipy, by default it tries to use up all available cores
-    time_limit: integer, maximum allowed running time, in seconds, default is 7200 (2 hour)
-    model_prefix: output prefix for gurobi *.lp model
-
-    Returns: (1) Status of gurobi optimization model (usually 2 - optimal; 3 - infeasible; 9 - suboptimal/reached time limit)
+    """Cycle decomposition by minimizing the number of cycles/paths.
+
+    Standard model for cycle detection used when generated BreakpointGraph is tractable.
+
+    Args:
+        amplicon_id: integer, amplicon ID
+        g: breakpoint graph (object)
+        k: integer, maximum mumber of cycles/paths allowed in cycle decomposition
+        total_weights: float, total length-weighted CN in breakpoint graph g
```

Comment:
All of these docstrings should have line length < 80 (put overflow on a new line with an indent)